### PR TITLE
Make Second Level Feedback strings localizable

### LINKED ISSUE #2474: Feedback categories should be localizable
#2419 replaced the feedback categories with new categories and subcategories but hard-coded the new strings as string literals. As a result, v1.0 would have a regression where the feedback UI would be unintelligible to speakers of languages other than English. The new strings should either be extracted into localizable strings or moved into a localizable property list.

/cc @mapbox/navigation-ios @nishant-karajgikar @avi-c

### PR DESCRIPTION
**Pull request by avi-c**
Move hard coded strings to localizable ones.

Fixes #2474.

### CODE REVIEW

File: MapboxNavigation/FeedbackItem.swift [hunk 1]
```diff
@@ -16,71 +16,71 @@ public extension FeedbackType {
         case .general:
             return "Feedback"
         case .incorrectVisual(.none):
-            return "Incorrect visual"
+            return NSLocalizedString("INCORRECT_VISUAL_FEEDBACK", bundle: .mapboxNavigation, value: "Incorrect visual", comment: "General category of route feedback where visual instruction was incorrect.")
```
Comment (1ec5) [inline MapboxNavigation/FeedbackItem.swift:19]: Can you make the same change to ”Looks Incorrect” as in https://github.com/mapbox/mapbox-navigation-ios/pull/2497#discussion_r462691238?

Comment (avi-c) [reply MapboxNavigation/FeedbackItem.swift]: Done.

File: MapboxNavigation/FeedbackSubtypeViewController.swift [hunk 2]
```diff
@@ -149,11 +149,12 @@ class FeedbackSubtypeViewController: FeedbackViewController {
...
+            reportButton.setTitle(NSLocalizedString("NAVIGATION_REPORT_CANCEL", bundle: .mapboxNavigation, value: "Cancel", comment: "Title for button that cancels user's submission of feedback on navigation session issues."), for: .normal)
+            
         } else if selectedItems.count == 1 {
-            reportButton.setTitle(NSLocalizedString("NAVIGATION_REPORT_ISSUE", comment: "Title for button that submits user's feedback on navigation session issues."), for: .normal)
+            reportButton.setTitle(NSLocalizedString("NAVIGATION_REPORT_ISSUE", bundle: .mapboxNavigation, value: "Send Item", comment: "Title for button that submits user's feedback on navigation session issues."), for: .normal)
         } else {
-            reportButton.setTitle(NSLocalizedString("NAVIGATION_REPORT_ISSUES", comment: "Title for button that submits user's feedback on multiple navigation session issues."), for: .normal)
+            reportButton.setTitle(NSLocalizedString("NAVIGATION_REPORT_ISSUES", bundle: .mapboxNavigation, value: "Send Items",  comment: "Title for button that submits user's feedback on multiple navigation session issues."), for: .normal)
```
Comment (1ec5) [inline MapboxNavigation/FeedbackSubtypeViewController.swift:157]: This logic doesn’t work in some of the supported languages, such as Russian. We use a stringsdict to enable more flexible localization of grammatical plurals and genders:

https://github.com/mapbox/mapbox-navigation-ios/blob/6e54f9d38bb09e7cb1cdac7d194364e69f7d3ea8/MapboxNavigation/RatingControl.swift#L105-L109 https://github.com/mapbox/mapbox-navigation-ios/blob/dc9f1e9be68b18ca4f57882c98e0a58e85461507/MapboxNavigation/Resources/en.lproj/Localizable.stringsdict#L21-L36

File: MapboxNavigation/FeedbackItem.swift [hunk 3]
```diff
@@ -16,71 +16,71 @@ public extension FeedbackType {
...
+            return NSLocalizedString("ROUTE_QUALITY_ROAD_CLOSURE_FEEDBACK", bundle: .mapboxNavigation, value: "Road closure", comment: "General route feedback that route contained closed road.")
         case .roadClosure(.streetPermanentlyBlockedOff):
-            return "Street permanently blocked off"
+            return NSLocalizedString("ROUTE_QUALITY_ROAD_CLOSURE_PERMANENT_FEEDBACK", bundle: .mapboxNavigation, value: "Street permanently blocked off", comment: "Specifc route feedback that route contained road that is permanently closed.")
         case .roadClosure(.roadMissingFromMap):
-            return "Road missing from map"
+            return NSLocalizedString("ROUTE_QUALITY_ROAD_MISSING_FROM_MAP_FEEDBACK", bundle: .mapboxNavigation, value: "Road missing from map", comment: "Specifc route feedback that route contained road not shown on map.")
         case .roadClosure(.other):
-            return "Other"
+            return NSLocalizedString("ROAD_CLOSURE_OTHER_FEEDBACK", bundle: .mapboxNavigation, value: "Other", comment: "Specific route feedback that a road closure type was encountered but not listed as a choice.")
```
Comment (1ec5) [inline MapboxNavigation/FeedbackItem.swift:83]: This was probably a pain to make localizable. As tail work, to reduce verbosity, we should move these strings into a localizable plist. That would also bring us a little closer to making the feedback categories more configurable.

File: MapboxNavigation/Resources/Base.lproj/Localizable.strings [hunk 4]
```diff
@@ -70,15 +85,57 @@
...
 "LESS_THAN" = "<%@";
 
+/* Title for button that cancels user's submission of feedback on navigation session issues. */
+"NAVIGATION_REPORT_CANCEL" = "Cancel";
+
+/* Title for button that submits user's feedback on navigation session issues. */
+"NAVIGATION_REPORT_ISSUE" = "Send Item";
+
+/* Title for button that submits user's feedback on multiple navigation session issues. */
+"NAVIGATION_REPORT_ISSUES" = "Send Items";
```
Comment (1ec5) [inline MapboxNavigation/Resources/Base.lproj/Localizable.strings:137]: Once we add the string format to Localizable.stringsdict, these two entries will collapse into a single entry, set to “Send Item(s)”, that will be overridden by the properly pluralized form if available.

**Pull request merged**